### ISSUE TITLE
Same action names for different entities lead to renamed interface name

### ISSUE THREAD

**Issue by heimwege** [odata2ts/odata2ts#256]
Hey hey,

I just re-generated the types based on the newest version 0.33.1 and there's one change that I don't understand. It seems to have been introduced in version 0.32.0.

Although the metadata did not change

```
...
<Action Name="draftPrepare" IsBound="true" EntitySetPath="in">
        <Parameter Name="in" Type="CapFeTsSampleService.Samples"/>
        <Parameter Name="SideEffectsQualifier" Type="Edm.String"/>
        <ReturnType Type="CapFeTsSampleService.Samples"/>
</Action>
<Action Name="draftPrepare" IsBound="true" EntitySetPath="in">
        <Parameter Name="in" Type="CapFeTsSampleService.Comments"/>
        <Parameter Name="SideEffectsQualifier" Type="Edm.String"/>
        <ReturnType Type="CapFeTsSampleService.Comments"/>
</Action>
...
```

I get the following console message

```
Duplicate name: draftPrepare - Fully Qualified Names: Samples_CapFeTsSampleService.draftPrepare, Comments_CapFeTsSampleService.draftPrepare (renamed to: draftPrepare2)
```

and the previously (odata2ts 0.31.1) generated interfaces 

```
export interface Samples_draftPrepareParams
export interface Comments_draftPrepareParams
```

are adjusted to

```
export interface Samples_draftPrepareParams
export interface Comments_draftPrepare2Params
```

(That's all the changes in the generated file. Nothing else)

As the interface name is anyway prefixed with `entityName_` there is no duplication and thus I'm a bit surprised by this change. Is this a bug?

**Comment by texttechne** [comment]
Hi @heimwege,

yeah, that's a bug.

**Comment by texttechne** [comment]
Hi @heimwege,

fixed with `@odata2ts/odata2ts` version `0.33.2`.
Thanks for the bug report!

Please close the issue if you can check that the issue has been fixed.

**Comment by heimwege** [comment]
Can confirm that the issue no longer occurs in `0.33.2`. Thanks for the fix 👍🏻
